02 Algorithm Reference/11 Universes/01 Introduction.html

```diff
@@ -10,5 +10,5 @@
 	Every algorithm has a hidden "user-defined-universe". The assets in this universe are set by the $[AddEquity] / $[AddForex] methods. These assets are fixed and never removed from your algorithm.
 </p>
 <p>
-	Universes (and data QuantConnect provides) are refreshed every day by default. Custom universes can be refreshed as often as required. This is controlled in the <code>algorithm.UniverseSettings</code> variable which we'll describe in more detail below.
-</p>
\ No newline at end of file
+	Universes (and data QuantConnect provides) are refreshed every day by default. Custom universes can be refreshed as often as required. This is controlled in the $[algorithm.UniverseSettings,T:QuantConnect.Data.UniverseSelection.UniverseSettings] variable which we'll describe in more detail below.
```

Comment:
Same as above: `UniverseSettings` is `P:QuantConnect.Algorithm.QCAlgorithm.UniverseSettings` instead of `T:QuantConnect.Data.UniverseSelection.UniverseSettings`, since we are talking about the helpers that are properties (P).